src/main.rs

```diff
@@ -805,9 +809,8 @@ fn main() {
                                 {
                                     // draw textured
                                     let uniforms = glium::uniform! {
-                                        model: <[[f32; 4]; 4]>::from(mat),
-                                        view: view_mat,
-                                        perspective: perspective_matrix,
+                                        norm_matrix: norm_matrix,
```

Comment:
I think you could just write `norm_matrix` by itself, rather than `norm_matrix: norm_matrix`.

Reply:
Normally, yes, if this were a normal struct, but this actually a macro which does require both tokens in this case.